### PR TITLE
fix: remove gitlab references

### PR DESCRIPTION
**Pull request by brenordr**
# Description

Remove references to Gitlab

Kind: Fix

Section:

Summary:

Issue link:

## Changelog

> List the main points of change

## Pay attention to

> Important parts of the code that require special attention

### CODE REVIEW

**Review by nicholaspcr — APPROVED** [review]

**Review by luiz-couto — APPROVED** [review]

File: docs/cli_install.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ sudo rm /usr/local/bin/inspr
 
 ## Other OS
 
-To get other Inspr CLI versions, download the binary file from the GitLab repository [here](https://github.com/inspr/inspr/-/releases).
+To get other Inspr CLI versions, download the binary file from the GitHub repository [here](https://github.com/inspr/inspr/-/releases).
```
Comment (ptcar2009) [inline docs/cli_install.md]: I think this link has to be changed.

**Review by ptcar2009 — CHANGES_REQUESTED** [review]

File: docs/cli_install.md (hunk 1 above)
Comment (brenordr) [reply docs/cli_install.md]: @ptcar2009 Why?

Comment (ptcar2009) [reply docs/cli_install.md]: I think there is no hyphen

**Review by brenordr — COMMENTED** [review]
Fixes applied with success

File: docs/cli_install.md (hunk 1 above)
Comment (brenordr) [reply docs/cli_install.md]: done

**Review by ptcar2009 — APPROVED** [review]
LGTM

**Pull request merged**